.changeset/olive-rocks-listen.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-splunk-on-call': minor
```

Comment:
Since this isn't a breaking change you can make this `patch`, we use that for non-breaking changes while we're on `0.x`.

Reply:
Ah, that makes sense! Thank you, just updated.